benchmark-compare.sh

```
@@ -158,25 +221,34 @@ check_jar_exists() {
+
+JVM_SYS_PROPS=""
+if [ -n "$BENCH_EDT_PATH" ]; then
+    JVM_SYS_PROPS="$JVM_SYS_PROPS -Dbench.edt.path=$BENCH_EDT_PATH"
+fi
+if [ -n "$BENCH_DESIGNER_PATH" ]; then
+    JVM_SYS_PROPS="$JVM_SYS_PROPS -Dbench.designer.path=$BENCH_DESIGNER_PATH"
+fi
+
 echo "📊 Запуск бенчмарков для прошлой версии..."
-java -jar "$RESULTS_DIR/old-version.jar" -prof com.github._1c_syntax.bsl.mdclasses.benchmark.MemoryProfiler -rf json -rff "$RESULTS_DIR/old-results.json"
+java $JVM_ARGS $JVM_SYS_PROPS -jar "$RESULTS_DIR/old-version.jar" $JMH_COMMON_ARGS $JMH_QUICK_ARGS -rf json -rff "$RESULTS_DIR/old-results.json"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)